src/strings.json

```diff
@@ -25,8 +25,8 @@
   "watchExpressions.header": "Watch Expressions",
   "watchExpressions.addButton": "Add Watch Expression",
   "watchExpressions.refreshButton": "Refresh",
-  "welcome.search": "%S to search for files",
-  "sourceSearch.search": "Search...",
```

Comment:
two things here:
1. we also need to change [properties](https://github.com/devtools-html/debugger.html/blob/master/assets/locales/debugger.properties)
2. I believe we need to use a different key so that L10N doesn't get confused